src/app/services/submissions.service.ts

```diff
@@ -1,15 +1,15 @@
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { HttpClient} from '@angular/common/http';
-import { map, flatMap } from 'rxjs/operators';
-
+import { RequestOptions } from '@angular/http';
```

Comment:
Please, do not use `@angular/http` it is deprecated. Use `@angular/common/http`

Reply:
OK, I fixed it